pkgs/tools/typesetting/xmlto/default.nix

```diff
@@ -64,6 +52,7 @@ stdenv.mkDerivation (finalAttrs: {
   # `libxml2' provides `xmllint', needed at build-time and run-time.
   # `libxslt' provides `xsltproc', used by `xmlto' at run-time.
   nativeBuildInputs = [
+    autoreconfHook
```

Comment:
infinite recursion...

Reply:
I wonder what system it fails the eval on and what attribute it is. All I try seems to work when merged to current `staging`:

```
$ for system in {x86_64,aarch64}-{linux,darwin}; do nix-instantiate -A xmlto --argstr system $system; done

/nix/store/gj6j9jycnvv2iz9n0byxmci36h4h7i8m-xmlto-0.0.29.drv
/nix/store/6263sr6ngy3gmqgqngjaksijjcdkyix0-xmlto-0.0.29.drv
/nix/store/645lhdwl34dnswlbnz12nfx71ir9ppb0-xmlto-0.0.29.drv
/nix/store/6xzlr847gmiw8kq54njdwkws40dwb0rb-xmlto-0.0.29.drv
```

Reply:
It fails through git. see the full ofborg log for the details

Reply:
Does it fail for you as well? What should I run locally to see the same failure?

Reply:
I didn't try it or locally but I would expect building git or gitFull would fail.

Reply:
I would expect `xmlto` itself to fail as well in that case. Tried to instantiate `git` and `gitFull` when merged this `PR` against `staging`. Instantiates just fine:

```
$ for system in {x86_64,aarch64}-{linux,darwin}; do nix-instantiate -A git -A gitFull --argstr system $system; done

/nix/store/nsbwv0n1hqjfmlfcivr7yrhimghxr8fm-git-2.45.2.drv
/nix/store/dybj21wa71zmv9iqn2v6zqc3xnb060cr-git-with-svn-2.45.2.drv

/nix/store/g0g8vpprs03ykd9jjdma0s575xjkdrvs-git-2.45.2.drv
/nix/store/wazdz8knx07ggyy56860xf9i3pb3k56v-git-with-svn-2.45.2.drv

/nix/store/qhjzf8qdcfshsiw6pz0r1blg34k385s5-git-2.45.2.drv
/nix/store/0dqaf7yhvqm1b57jhn4nrg21k89b59r4-git-with-svn-2.45.2.drv

/nix/store/xw734k1v52dhac1mk67psimzw415rhdp-git-2.45.2.drv
/nix/store/0ih7aa39fmw4szicr46g103l4li83a02-git-with-svn-2.45.2.drv
```

I suspect the eval failure was done against some stale state of `staging`.

Reply:
> I suspect the eval failure was done against some stale state of staging.

That seems to have been the root cause.